src/crypto/elliptic/p256_asm_table.go

```diff
@@ -0,0 +1,1473 @@
+// Copyright 2021 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build amd64 || arm64
+// +build amd64 arm64
+
+package elliptic
+
+var p256Precomputed = &[43][32 * 8]uint64{
```

Comment:
Is this a new 88 KB that could be const?

Reply:
Reverted for now. This was introduced in 1.17, so I pulled it back out. Will fix properly when we upgrade our toolchain to 1.17.